src/examples/Colors/ColorsTemplate.js

```diff
@@ -7,437 +7,77 @@ const ColorBubble = ({colorClass}) => {
     );
```

Comment:
can you revert these?

Reply:
Done

Reply:
Is there a challenge here or you meant something else